manifests/service.pp

```diff
@@ -24,7 +24,7 @@
     ],
   }
 
-  if $confluence::manage_service {
+  if str2bool($confluence::manage_service) {
```

Comment:
str2bool is not needed. Instead, add the `Boolean` type for the `manage_service` parameter

Reply:
fixed